tests/unit/projection-test.js

```diff
@@ -1307,6 +1286,7 @@ module('unit/projection', function(hooks) {
 
       run(() => {
         set(projectedExcerpt, 'publisher.location', NEW_PUBLISHER_LOCATION);
+        set(projectedExcerpt, 'publisher.owner', NEW_PUBLISHER_OWNER);
```

Comment:
I believe this should be an expect throws and the counts below should reflect that owner does not change since the publisher projection does not allow `owner`.

Reply:
That's fair, didn't look at whether we have failing `set` for resolved entity.

Reply:
I mistook excerpt here for preview, this set should not be expected to throw